client/src/document/toolbar/flaws.tsx

```diff
@@ -133,6 +133,13 @@ function BrokenLinks({
 
   const filepath = sourceFolder + "/index.html";
 
+  function equalUrls(url1: string, url2: string) {
```

Comment:
wouldn't something like `isSamePathname` or even `isSameSlug` be a more precise name? I guess a slug is a kind of URL, so this is very nitty

Reply:
Yeah, I think you're right. But, note that the function is inlined so its name isn't nearly as important than if it was standing on its own two legs as a general utility function. 
And I can think of flaws with the name "isSamePathname" too because you'd expect to be passed two pathnames then as arguments. Why is naming so hard! Even after so many years!

Based on the fact that the nit isn't a slam dunk I'm going to boldly go ahead with this one for now. Thank you for taking the time to bringing it up.